### PR TITLE
Fixed #3984

### LINKED ISSUE #3984: Item 12399 missing script
* **rAthena Hash**: 

* **Client Date**:  Not relevant

* **Server Mode**: Not relevant

* **Description of Issue**: Item 12399, treasure chest is a consumable item that is supposed to produce zeny when used, obtainable from woe TE castles. It has no script so it does not produce the zeny.
  * Result: 
  * Expected Result: 
  * How to Reproduce: 
  * Official Information: https://www.divine-pride.net/database/item/12399/kRO 
  

* **Modifications that may affect results**: 
  
   None

### PR DESCRIPTION
**Pull request by BrOgBr**
* **Addressed Issue(s)**:  #3984

* **Server Mode**:  Pre-Renewal and Renewal

* **Description of Pull Request**:  Fixed missing item script 12399. `Gives 1000000 to 1000000 zeny when used`

Thanks @SeravySensei and Divine Pride.

### CODE REVIEW

**Comment by secretdataz** [comment]
Is a WoE:TE castle reward suppose to exist in pre-renewal?

File: db/re/item_db.txt [hunk 1]
```diff
@@ -6626,7 +6626,7 @@
...
 12397,Fools_Day_Box2,Gift Box?,11,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ .@rnd = rand(1,10); if(.@rnd==1) itemskill "TF_DETOXIFY",1; else if(.@rnd==2) itemskill "TF_PICKSTONE",1; else if(.@rnd==3) itemskill "BA_FROSTJOKER",1; else if(.@rnd==4) itemskill "DC_SCREAM",1; else if(.@rnd==5) end; else if(.@rnd==6) getitem 909,1; else if(.@rnd==7) itemskill "AL_RUWACH",1; else if(.@rnd==8) specialeffect2 EF_BEGINASURA; else if(.@rnd==9) specialeffect2 EF_MVP; else specialeffect2 EF_CURSEATTACK; },{},{}
 12398,PCBang_Gift_Box,PCRoom Gift Box,2,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{},{},{}
-12399,Castle_Treasure_Box,Castle Treasure Box,2,20,,1000,,,,,0xFFFFFFFF,63,2,,,,,,{},{},{}
+12399,Castle_Treasure_Box,Castle Treasure Box,2,20,,1000,,,,,0xFFFFFFFF,63,2,,,,,,{ Zeny += rand(1000000,1000000); },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: `Zeny += 1000000;`

**Comment by BrOgBr** [comment]
> Is a WoE:TE castle reward suppose to exist in pre-renewal?
The item is in pre-renewal db.

File: db/re/item_db.txt (hunk 1 above)
Comment (BrOgBr) [reply db/re/item_db.txt]: Thanks Atemo, I'll fix.

**Pull request merged**